testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,118 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8 import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):
+
+    '''Unittests for RearrangeCars class and promised fuctionality
+    '''
+
+    def setUp(self):
+
+        '''Basic setting that will be used in several tests
+        '''
+        
+        self.initial_car_order = [1, 2, 0, 3]
+        self.final_car_order = [3, 1, 2, 0]
+        self.rc = RearrangeCars(list(self.initial_car_order), list(self.final_car_order))

```

Comment:
@muzaluisa @yarynapavlyshyn So to sum it all up... I listed them so that if these variables were modified by calling out `rearrange_all_cars`, we will be able to detect them. i.e. without the listing, the `test_initial_order_preserved` wouldn't make much sense, since the variables that we are asserting equal to will be the same variable...